packages/terra-date-picker/tests/jest/__snapshots__/DatePickerField.test.jsx.snap

```diff
@@ -1938,6 +1937,7 @@ exports[`should render a valid DatePickerField with props 1`] = `
           >
             Label Test
             <div
+              aria-hidden="true"
```

Comment:
> I see you have updated a couple of jest snapshots for terra-date-picker which are unrelated to the changes you intend to make in this PR. Looks like the main branch of your forked repo is behind the actual main. Can you pull in the latest code and make sure the date picker changes are reverted.

@saket2403 - after reviewing further, it looks like the current main branch also has these same failures in the jest snapshot for DatePickerField due to these two recent updates from terra-core in the TerraFormField:
- [../terra-core/pull/3687: **Removing aria-live="assertive" from error messages**](https://github.com/cerner/terra-core/pull/3687/files#diff-84d6aefc84bd82a4f9a1fdedcebe5d2642539a25f9e41eb918ed87cf46f70b77L205)
- [../terra-core/pull/3686: **Making required field asterisk as aria-hidden="true"**](https://github.com/cerner/terra-core/pull/3686/files#diff-84d6aefc84bd82a4f9a1fdedcebe5d2642539a25f9e41eb918ed87cf46f70b77R185)

Normally we'd probably would have caught it after a build failure following the latest terra-core release and do the fix as it's own PR.

Reply:
In that case we can merge these changes with this PR.